src/index.html

```diff
@@ -2,12 +2,233 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <title>Title</title>
+    <meta name="viewport"
+          content="width=device-width,
+          user-scalable=no,
+          initial-scale=1.0,
+          maximum-scale=1.0,
+          minimum-scale=1.0">
+    <title>Uber Eats</title>
     <link rel="stylesheet" href="./styles/main.css">
   </head>
-  <body>
-    <img src="./images/logo.svg" alt="Logo">
-    <h1>Hello Mate Academy</h1>
+  <body class="body">
+    <header>
+      <div class="header">
+        <div class="container header__container">
+          <a href="#" class="header__logo logo_link">
+            <img src="./images/logo.svg" alt="uber eats">
+          </a>
+          <div class="asap">
+            <label class="asap__label">
+              <span>When</span>
+              ASAP
+              <input type="checkbox">
+            </label>
+            <label class="label">
+              <span>To</span>
+              <input class="input asap__input" type="text">
+            </label>
+          </div>
+          <div class="header__signin">
+            <button class="button button--white" type="button">Sign In</button>
+            <button class="button button--black" type="button">Register</button>
+            <button class="button button--img button--backet" type="button">1</button>
+          </div>
+        </div>
+      </div>
+      <div class="container">
+        <input class="input__search" type="text" placeholder="Search for restaurant or cuisine">
+      </div>
+    </header>
+
+    <main class="main">
+      <div class="container">
+        <section class="city-group">
+          <h1 class="city-group__title">Kyiv Restaurants</h1>
+          <div class="card-list">
+            <div class="card-list__item">
+              <div class="card-img">
+                <img src="./images/goods/macdonalds.png" alt="Макдоналдс">
```

Comment:
```suggestion
                <img src="./images/goods/macdonalds.png" alt="mcdonalds photo">
```
И все последующие alt для картинок